src/assets/sass/website/containers/about.scss

```diff
@@ -0,0 +1,118 @@
+@import "../color-palette.scss";
+@import "../variables.scss";
+@import "../mixins.scss";
+@import "../shared/half-clipped-shape.scss";
+
+.about-wrapper {
+  background-color: $primary;
+  height: max-content;
+  margin-top: -$header-height;
+  min-height: calc(100vh - #{$footer-height});
+  @include is-mobile {
+    margin-top: -$header-height-mobile - 45;
+    min-height: calc(100vh - #{$footer-height-mobile});
+  }
+  @media (max-width:600px) {
+    margin-top: -75px;
+  }
+  .hero-wrapper{
+    @media (max-width:600px) {
```

Comment:
create a variable for `600px` at the top of the file or in `variables.scss` if u use it in other files